src/app/modules/item/components/item-children-edit/item-children-edit.component.ts

```diff
@@ -31,18 +32,19 @@ export class ItemChildrenEditComponent implements OnChanges {
       this.subscription?.unsubscribe();
       this.subscription = this.getItemChildrenService
         .get(this.itemData.item.id, this.itemData.currentResult.attemptId)
-        .subscribe(children => {
-          this.data = children.map((child, idx) => ({
+        .pipe(
+          map(children => children.map((child, idx) => ({
             id: child.id,
             title: child.string.title,
             order: idx
```

Comment:
Ah I didn't see it earlier: the order has to be the order returned by the service!

Reply:
Fixed